src/assets/style/module/info/_info.less

```diff
@@ -0,0 +1,27 @@
+#airportInfo {
+    color: @light-gray;
+    font-size: 12px;
+    left: 0;
+    padding: 8px;
+    position: absolute;
+    top: 0;
+    width: auto;
+    z-index: 1000;
+}
+
+.airportInfo-label {
+    width: 35px;
+    float: left;
```

Comment:
why not `flexbox`?

Reply:
Now that you mentioned it...I tend to use table here instead (because that's what it is).

What would be the benefit of using flexbox?